addons/stock/models/stock_move.py

```diff
@@ -647,18 +647,30 @@ def action_assign_serial(self):
 
     def _do_unreserve(self):
         moves_to_unreserve = self.env['stock.move']
+        moves_not_to_recompute = self.env['stock.move']
```

Comment:
probably need to delete code in mrp https://github.com/odoo/odoo/blob/dbab68a92ec48b2bb6f21244af1de56691cb43f1/addons/mrp/models/stock_move.py#L194